### PR TITLE
Add Ulysses Extension

### PR DESCRIPTION
**Pull request by praveenjuge**
## Description

This is an extension for [Ulysses](http://ulysses.app) app. Currently it has one command to open a new new empty sheet in Ulysses.

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/13696888/149926435-f81d8cf9-b094-421e-bf74-9d48bb138932.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/ulysses/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "name": "ulysses",
+  "title": "Ulysses",
+  "description": "Use Ulysses with Raycast",
+  "icon": "command-icon.png",
+  "author": "praveenjuge",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "new-empty-sheet",
+      "title": "New Empty Sheet",
```
Comment (pernielsentikaer) [inline extensions/ulysses/package.json:12]:
```suggestion
      "title": "New Empty Sheet",
      "subtitle": "Ulysses",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @praveenjuge 👋 

Only a single correction before its ready 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi @praveenjuge 👋 

Thanks for your contribution 💯 I'm going to accept it and then will my colleague do a final review before it's finally merged.

**Review by mattisssa — CHANGES_REQUESTED** [review]
Hey! Thanks for the contribution.

For some reason, I got the error, but as you can see I have the app installed and even opened:

CleanShot 2022-01-20 at 13 53 59@2x

And I also don't think the error has a nice text description showing the details. Probably something like "Install it from: https:..." would be better. 

I'm not sure if something wrong with my set up, didn't do anything special. Let me know how can I help

**Comment by praveenjuge** [comment]
@mattisssa I'll take a look on what might have on wrong and update the error message 👍

**Comment by pernielsentikaer** [comment]
Strange, I had no issues at all, it opened the app just fine without any problems at all here

**Comment by mattisssa** [comment]
@pernielsentikaer @praveenjuge 
>➜  ~ mdls -name kMDItemCFBundleIdentifier -r /System/Volumes/Data/Applications/Setapp/Ulysses.app                                              
com.soulmen.ulysses-setapp% 

A good point I have is that I installed it from [Setapp](https://setapp.com), and the Ulysses is located in the folder inside Setapp one.

**Comment by pernielsentikaer** [comment]
Makes sense, now it should be easier for @praveenjuge to adjust 😄

**Comment by praveenjuge** [comment]
Yes, I will adjust it to see look from both app store and set app version 👍

**Comment by pernielsentikaer** [comment]
Sounds like the right plan, awesome! That and the suggestions for the toast, and we're about being there!

**Review by pernielsentikaer — APPROVED** [review]
Hi @praveenjuge 

Much better, should be good to go now 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by mattisssa — APPROVED** [review]
Now it works, thanks for adding Setapp's bundle 👏 
@pernielsentikaer Thanks for pinging 🚀

**Pull request merged**